src/lib/features/client-feature-toggles/delta/client-feature-toggle-delta.ts

```diff
@@ -171,26 +155,48 @@ export class ClientFeatureToggleDelta {
         if (!hasSegments) {
             await this.updateSegments();
         }
-
         if (requiredRevisionId >= this.currentRevisionId) {
             return undefined;
         }
+        if (requiredRevisionId === 0) {
```

Comment:
Might be missing some context, but if this is the revisionId coming from the SDK, couldn't it possibly be undefined? Why a strict check on 0?

Reply:
https://github.com/Unleash/unleash/pull/9133/files#diff-117921481602afc7d7b53fef3f43e7c22c8e08d938f28512444b1021bdcb86c5R145 normalizes it.